raw_section_snooper_vocab.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+"""
+    raw_section_snooper - looks for ANY sections (XML elemnent tagged "section"?),
+        within each type:
+        - entry: ;shows template Id, concepts and names if present,
+        - entry/organizer: ;shows template Id, concepts and names if present,
+        - title
+        - code
+        - value TBD
+
+"""
+
+"""
```

Comment:
If the point of this code is to prepare data for JHU to create the vocabulary crosswalk, we don't need tag and template id. What I was rambling on about in the issue ticket, was how we might do this with two scripts: one for the coverage report where we need the tag and section detail, and one for creating the vocabulary crosswalk where we need the whole analysis in one file and merged...without tag and section.